config/settings.yml

```diff
@@ -89,6 +89,16 @@
     :purge_schedule: "50 * * * *"
     :purge_window_size: 10000
     :queue_timeout: 20.minutes
+docs:
+  :ansible_tower_provider: https://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#adding-an-ansible-tower-provider
```

Comment:
naming is hard... it seems ok to me though

Reply:
Do we have hardcoded code looking for the `ansible_provider` key?  If so, does it make sense to do the move in one PR here and then drop it from UI-classic and change all of the `ansible_provider` key references to whatever we decide in a subsequent PR?

Reply:
This is just a move for now from UI... this particular key is definitely wrong :)

Reply:
Right, currently it's changing 
`:ansible_provider` to `:ansible_tower_provider`

so, I'm concerned there are references in the ui-classic to the old `ansible_provider` key...

/me looks...

like this: https://github.com/ManageIQ/manageiq-ui-classic/blob/dd276246da02adec0f69f4517b0ad899591d3a50/app/views/automation_manager/explorer.html.haml#L26

Reply:
Ah, the description now has the link to that change.  👍